core/src/trezor/ui/layouts/common.py

```diff
@@ -26,21 +26,20 @@ async def button_request(
         log.debug(__name__, "ButtonRequest.type=%s", br_type)
     workflow.close_others()
     if pages is not None:
-        await ctx.call(ButtonRequest(code=code, pages=pages), ButtonAck)
+        await context.maybe_call(ButtonRequest(code=code, pages=pages), ButtonAck)
     else:
```

Comment:
Instead of this if-else, just single `await context.maybe_call(ButtonRequest(code=code, pages=pages), ButtonAck)` should be enough

Reply:
81305badef3cbadaa7de0b696ce83953dae4089f